index.js

```diff
@@ -1,10 +1,20 @@
+const bulk = require('bulk-require') 
+const path = require('path')
+
 // polyfills
 require('setimmediate')
 
 require('depject')(
   // from more specialized to more general
-  require('./modules_extra'),
-  require('./modules_basic'),
-  require('./modules_core')
+  load('modules_extra'),
+  load('modules_basic'),
+  load('modules_core')
 ).app[0]()
 
+
+function load (moduleSetName) {
```

Comment:
yeah I know this is a convenience method.
I spend an hour on brfs + bulkify playing well together and did all the things down to the point of hard-coding addresses (removing alllllllllll fanciness). NOPE.

I figured I would loop back around once there was more clarity about this approach. Worst case can re-introduce the `index.js` files hand-written